Tests/AppTests/Extensions/Foundation/String/String+Random.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+
+extension String {
+  static var random: String {
+    let allowedChars = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
```

Comment:
Why don't use CharacterSet alphanumerics? One another way for create random string is give md5 for different random numbers.

Reply:
I didn't use md5, because it is with fixed length. So I moved to `uuid` generator and substring it with random number.